docs/UserGuide.md

```diff
@@ -226,15 +226,15 @@ A volunteer can have any number of [tags](#tags), [medical qualifications](#medi
 Examples:
 
 * `add_volunteer n/John Wick bd/1990-11-05 e/johnwick@example.com a/New yourk p/99887432 ic/S9054567D` adds a volunteer named `John Wick` with [NRIC](#nric)
-  `S9054567D` to FriendlyLink, with [birth date](#birthdate) `1990-11-05`, [address](#address) `New yourk`, [email address](#email) `johnwick@example.com` and [phone number](#phone-number) `99887432`.
+  `S9054567D` to FriendlyLink, with [birth date](#birthdate) `1990-11-05`, [address](#address) `New york`, [email address](#email) `johnwick@example.com` and [phone number](#phone-number) `99887432`.
```

Comment:
His address is New yourk. Pls change the top part too if you want it to follow the real place 😄